internal/pkg/vm/actor/builtin/miner/miner.go

```diff
@@ -192,6 +114,38 @@ type State struct {
 	OwedStorageCollateral types.AttoFIL
 }
 
+// Ask is a price advertisement by the miner
+type Ask struct {
+	Price  types.AttoFIL
+	Expiry *types.BlockHeight
+	ID     *big.Int
+}
+
+const (
+	AddAsk types.MethodID = iota
+	GetOwner
+	CommitSector
+	GetWorker
+	GetPeerID
+	UpdatePeerID
+	GetPower
+	AddFaults
+	SubmitPoSt
+	SlashStorageFault
+	ChangeWorker
+	VerifyPieceInclusion
+	GetSectorSize
+	GetAsks
+	GetAsk
+	GetLastUsedSectorID
+	GetProvingSetCommitments
+	IsBootstrapMiner
+	GetPoStState
+	GetProvingWindow
+	CalculateLateFee
+	GetActiveCollateral
```

Comment:
When we come to align method ids with the spec, we'll probably want to specify the value of each of these constants explicitly, rather than rely on iota magic.

Reply:
When they get specified, agreed. 
https://github.com/filecoin-project/specs/issues/616